### ISSUE TITLE
Kotlinx Serialization Protobuf Generator

### ISSUE THREAD

**Issue by marcoferrer** [marcoferrer/kroto-plus#21]
This is the tracking issue for https://github.com/marcoferrer/kroto-plus/pull/13
 Originally work on this feature was blocked due to a bug with the ```@SerialInfo``` annotation in the  kotlin serialization native runtime. It looks like the bug is addressed in versions ```0.10.0-eap-1``` and ```1.3.20-eap-52``` with an expected release in ```1.3.20```

**Comment by cretz** [comment]
Just as a heads up, I have another library that does this at https://github.com/cretz/pb-and-k. Just figured I'd toss it out there though I understand reasons for sticking with Kotlinx.

**Comment by marcoferrer** [comment]
I was actually interested in adding support for your library as a serialization format for the grpc service generation. I needed to get some time to look into it. Providing support for kotlinx was something I wanted to do for a while, so this was just me following through with it.

**Comment by RdeWilde** [comment]
Development of pb-and-k has been slowed down due to the original developer having limited time. Therefore I think MP support for this project is something many would like.

**Comment by cretz** [comment]
> Development of pb-and-k has been slowed down due to the original developer having limited time. 

This is true...apologies, much of my day job has shifted away from Kotlin.

**Comment by RdeWilde** [comment]
> (..) It looks like the bug is addressed in versions `0.10.0-eap-1` and `1.3.20-eap-52` with an expected release in `1.3.20`

We're past `1.3.20` \o/

**Comment by RdeWilde** [comment]
Was there already work done on the kotlinx integration? @marcoferrer
